docs/advanced/pcidevices.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 0
+sidebar_label: PCI Devices
+title: ""
+---
+
+# PCI Devices
+
+A PCIDevice represents a host device with a PCI address. 
+The devices can be passed through the hypervisor to a VM by creating a PCIDeviceClaim, 
+or by using the UI to enable passthrough. Passing a device through the hypervisor means that 
+the VM can directly access the device, and effectively owns the device. A VM can even install 
+it's own drivers for that device.
+
+## Enabling passthrough on a PCI Device
+
+Go to the Advanced -> PCI Devices UI:
+
+![](/img/v1.1/pcidevices/advanced-pcidevices-index.png)
+
+Search for your device by vendor name (e.g. NVIDIA, Intel, etc.) or device name.
+
+![](/img/v1.1/pcidevices/search-pcidevices.png)
+
+Select the devices you want to enable for passthrough:
+
+![](/img/v1.1/pcidevices/select-pcidevices.png)
+
+Then click "Enable Passthrough", and wait until all the devices say "Enabled":
```

Comment:
@tlehman, we'll add a warning in GUI (https://github.com/harvester/harvester/issues/3003). Could we also add a warning here? Letting users know passing through the wrong device can damage the node.